include/TBetaGenerator.hpp

```diff
@@ -0,0 +1,363 @@
+// Collect all physical and utility constants in one specific place
```

Comment:
@yramachers As ever, I'm torn between directly including the code (ease of use for users, no forgetting of `--recursive` flag) vs including as a submodule. For now I'm going with direct inclusion, but I'm happy to discuss/entertain possible alternative viewpoints.